packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1540,8 +1546,9 @@ export class DeltaManager
                 },
                 cacheOnly);
         } catch (error) {
-            this.logger.sendErrorEvent({eventName: "GetDeltas_Exception"}, error);
-            this.close(CreateContainerError(error));
+            const eventName = "GetDeltas_Exception";
+            this.logger.sendErrorEvent({ eventName }, error);
+            this.close(eventName, CreateContainerError(eventName, error));
```

Comment:
I don't think your args for `CreateContainerError` are correct... And too many `any`'s for the compiler to help us out! 😱

Reply:
Yep, that's copy paste error